src/ParserFactory.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * This file is part of the Ray.Aop package
+ *
+ * @license http://opensource.org/licenses/bsd-license.php BSD
+ */
+namespace Ray\Aop;
+
+use PhpParser\Lexer;
+use PhpParser\Parser;
+use PhpParser\ParserFactory as PHPParserFactory;
+
+class ParserFactory
+{
+    public static function create()
```

Comment:
I know it's not a big deal, But I prefer using `(new ParserFactory )->newInstance()` here to have a consistency in entire our code base.